content/sensu-go/5.18/reference/agent.md

```diff
@@ -1437,9 +1438,11 @@ $ sudo systemctl restart sensu-agent
 
 {{< /language-toggle >}}
 
-## Use environment variables with the Sensu agent
+### Use environment variables with the Sensu agent
 
-After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to check and hook commands executed by the Sensu agent.
+After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), you can create custom environment variables there (in addition the environment variables Sensu already recognizes, such as those listed in the [agent configuration tables][16]).
```

Comment:
This reads a bit odd to me, but maybe I am just not understanding it all the way. Actually even the previous language was a bit confusing.

I read this as "after you configure your sensu-agent service to read environment variables....you can then create environment variables there." Where is there? Is this saying one thing, then saying the same thing again?

Reply:
Yes, I see what you're saying, thank you. Let me try to clarify...